contrib/TestHarness2/test_harness/run.py

```diff
@@ -123,7 +123,7 @@ def load_stats(self, serialized: str):
         times.frombytes(base64.standard_b64decode(serialized))
         assert len(times) == len(self.tests.items())
         idx = 0
-        for _, spec in self.tests.items():
+        for idx, (_, spec) in enumerate(self.tests.items()):
```

Comment:
This should also remove the `idx` used in line 125 and 128 (`idx = 0` and `idx += 1`)

Reply:
yes, already did (on my local disk -- will push as soon as I am through all your comments